CONTRIBUTING.md

```diff
@@ -25,49 +25,46 @@ all your interactions with the project members and users.
 
 ## Pull Requests (PRs)
 
-### Process
-1. Essential bug fix PRs should be sent to both master and devopment branches.
-2. Small bug fix and feature enhancement PRs should be sent to development only.
-3. Follow the existing code style precedent. This does not need to be strictly
-   defined as there are many thousands of lines of examples. Note the lack
-   of tabs anywhere in the project, parentheses and spacing, curly bracket
-   locations, source code layout, variable scoping, etc. and follow the
-   project's standards.
+1. Essential bug fix PRs should be sent to both master and release branches.
```

Comment:
It's nice if people do this, but I think it's actually fine to just send them to the release branch and then we will cherry pick them into master.  Whatever.